### PR TITLE
GeoTools 21.0 release script updates

### PR DESCRIPTION
**Pull request by jodygarnett**
Fixing and automating:

* [x] Update rename.xml to generate date and substitute correct doc target into GeoTools.html
* [x] Update GeoTools.html with Java 11 and Java 8 compatibility
* [x] Adjust artifact downloads to include license
* [x] Move LGPL license into place to be picked up by GitHub
* [x] Removed GPL+CP license as no longer in use
* [x] Added CCbyA license for inclusion in documentation build
* [x] dynamically generate (c) date for sphinx docs
* [x] updated artifact bundles so post-processing by shell script not required (to remove .gitfolder)
* [x] removed outdated build support for subversion and eclipse formatting

### CODE REVIEW

File: README.html [hunk 1]
```diff
@@ -3,27 +3,16 @@
...
 	<META HTTP-EQUIV="Content-Style-Type" CONTENT="text/css">
 	<STYLE TYPE="text/css">
 	<!--
-		H1.western { font-family: "Liberation Sans", sans-serif }
-		H2.cjk { font-family: "WenQuanYi Micro Hei" }
-		H2.ctl { font-family: "Lohit Hindi" }
-		H3.western { font-family: "Liberation Sans", sans-serif }
-		H3.cjk { font-family: "WenQuanYi Micro Hei" }
-		H3.ctl { font-family: "Lohit Hindi" }
+		H1 { font-family: "Liberation Sans", sans-serif }
```
Comment (tbarsballe) [inline README.html]: I do wonder if some of these are important for any non-english text (perhaps links - do we link to translations of the docs here?)?

File: build/rename.xml [hunk 2]
```diff
@@ -18,6 +18,7 @@
     <replace file="../README.html">
       <replacefilter token="@RELEASE@" value="@VERSION@"/>
       <replacefilter token="@DATE@" value="@RELEASE_DATE@"/>
+      <replacefilter token="@STABLE@" value="@SERIES@">
```
Comment (tbarsballe) [inline build/rename.xml]: Shouldn't this be `@LINK@`?

Comment (jodygarnett) [reply build/rename.xml]: Yes

File: README.html (hunk 1 above)
Comment (jodygarnett) [reply README.html]: No this is just boilerplate generated by LibreOffice

**Comment by jodygarnett** [comment]
Applying to master as 41633b24c7b2b94f4559af45abe6a88c3c515960

**Comment by jodygarnett** [comment]
Applying to maintenance as 37cd3fae922102d18a881b54ddb30d6d240c0ba2

**Pull request merged**